src/DesignCompile/CompileHelper.cpp

```diff
@@ -2530,8 +2530,6 @@ bool CompileHelper::isMultidimensional(UHDM::typespec* ts,
     } else if (ttps == uhdmbit_typespec) {
       bit_typespec* lts = (bit_typespec*)ts;
       if (lts->Ranges() && lts->Ranges()->size() > 1) isMultiDimension = true;
-    } else if (ttps == uhdmstruct_typespec) {
```

Comment:
This is intentional to prevent reduction to happen. Please leave it the way it was.